marlowe-playground-client/src/Examples/JS/Contracts.purs

```diff
@@ -7,9 +7,6 @@ const alice : Party = role("alice");
 const bob : Party = role("bob");
```

Comment:
I know this was already like this but why is this a purs file? could we save it as a ts file and embed it somehow instead.